executable/swf.ksy

```diff
@@ -104,7 +110,94 @@ types:
             'tag_type::script_limits': script_limits_body
             'tag_type::symbol_class': symbol_class_body
             'tag_type::set_background_color': rgb
+            'tag_type::define_sprite': define_sprite_body
             'tag_type::export_assets': symbol_class_body
+            'tag_type::define_shape4': define_shape4_body
+  define_shape4_body:
+    seq:
+      - id: id
+        -orig-id: ShapeId
+        type: u2
+      - id: shape_bounds
+        type: rect
+      - id: edge_bounds
+        type: rect
+      - id: reserved
+        type: b5
+      - id: use_fill_winding_rule
+        type: b1
+      - id: use_non_scaling_stroke
+        type: b1
+      - id: use_scaling_stroke
+        type: b1
+      - id: shapes
+        type: shape_with_style
+  shape_with_style:
+    seq:
+      - id: fill_styles
+        type: fill_style_array
+  fill_style_array:
+    seq:
+      - id: fill_style_count
+        type: u1
+      - id: fill_style_count_extended
+        type: u2
+        if: fill_style_count == 0xFF
+      - id: fill_styles
+        type: fill_style
+        repeat: expr
+        repeat-expr: fill_style_count
+  fill_style:
+    seq:
+      - id: fill_style_type
+        type: u1
+        enum: fill_style_type_enum
+      - id: color_a
```

Comment:
Plese add some words (and maybe change `id`s) about how these 2 colors are different and why we need both.

Reply:
It is still unclear what it means.